locksmith/config/config.ts

```diff
@@ -1,6 +1,13 @@
+import Stripe from 'stripe'
+
 require('dotenv').config()
 
-module.exports = {
+const stripeConfig: Stripe.StripeConfig = {
```

Comment:
This file was moved to TS so we could type this config (otherwise it complained that the type 'string' for `apiVersion` was too general).

It's now required that we pass a `Stripe.StripeConfig` when instantiating. See: https://github.com/stripe/stripe-node#usage-with-typescript